### PR TITLE
[TEST] Remove duplicated main response unit test

### PR DESCRIPTION
**Pull request by javanna**
The `toXContent` output of `MainResponse` was tested in two different test classes, unified tests in the existing `MainResponseTests`

### CODE REVIEW

File: core/src/test/java/org/elasticsearch/action/main/MainResponseTests.java [hunk 1]
```diff
@@ -51,6 +53,20 @@ public static MainResponse createTestItem() {
...
+        BytesStreamOutput streamOutput = new BytesStreamOutput();
+        mainResponse.writeTo(streamOutput);
+        final MainResponse serialized = new MainResponse();
+        serialized.readFrom(streamOutput.bytes().streamInput());
+
+        assertThat(serialized.getNodeName(), equalTo(mainResponse.getNodeName()));
+        assertThat(serialized.getClusterName(), equalTo(mainResponse.getClusterName()));
+        assertThat(serialized.getBuild(), equalTo(mainResponse.getBuild()));
+        assertThat(serialized.isAvailable(), equalTo(mainResponse.isAvailable()));
+        assertThat(serialized.getVersion(), equalTo(mainResponse.getVersion()));
```
Comment (tlrx) [inline core/src/test/java/org/elasticsearch/action/main/MainResponseTests.java]: Can we use MainResponse's equals() method instead? That would also check the cluster UUID

**Review by tlrx — APPROVED** [review]
LGTM, left a comment

File: core/src/test/java/org/elasticsearch/action/main/MainResponseTests.java (hunk 1 above)
Comment (javanna) [inline core/src/test/java/org/elasticsearch/action/main/MainResponseTests.java]: good one, I am also looking into migrating the whole class to some other existing base class like `AbstractStreamableXContentTestCase`, just found out it exists :)

**Comment by javanna** [comment]
With some of my recent changes I ended up stepping on @colings86 's toes I think. Sorry! I added you as a reviewer.

**Review by colings86 — APPROVED** [review]

**Pull request merged**